maquetacion/src/scss/_navbar.scss

```diff
@@ -0,0 +1,29 @@
+@import 'variables';
+
+.nav {
+  background-color: $light;
+  border-top: 6px solid $wolox-blue;
+  box-shadow: 0 4px 10px 0 $half-black;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+  
+  &__content {
```

Comment:
I think this is not needed, you can use the flexbox attributes in the `.nav` class